recipes/presidio-analyzer/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "presidio-analyzer" %}
+{% set version = "2.2.355" %}
+{% set python_min = "3.9" %}
+{% set python_max = "4.0" %}
```

Comment:
The min is set based on Python EOL or above, if the package requires it. You only need to define it if it is above. You don't have to create a jinja variable for the max, it only obscures the recipe.

```suggestion
```